oggsound/src/OgreOggSoundManager.cpp

```diff
@@ -360,10 +360,10 @@ namespace OgreOggSound
 		// If no manager specified - grab first one 
 		if ( !scnMgr )
 		{
-			Ogre::SceneManagerEnumerator::SceneManagerIterator it=Ogre::Root::getSingletonPtr()->getSceneManagerIterator();
+			Ogre::SceneManagerEnumerator::Instances inst = Ogre::Root::getSingletonPtr()->getSceneManagers();
```

Comment:
```suggestion
			auto& inst = Ogre::Root::getSingletonPtr()->getSceneManagers();
```
otherwise we create a copy here

Reply:
Done